Cargo.toml

```diff
@@ -86,7 +86,8 @@ imsz = "0.2"
 indexmap = { version = "2.10.0", features = ["std"] }
 ipc-channel = "0.20"
 itertools = "0.14"
-js = { package = "mozjs", git = "https://github.com/servo/mozjs" }
+#js = { package = "mozjs", git = "https://github.com/servo/mozjs" }
+js = { package = "mozjs", git = "https://github.com/Taym95/mozjs", branch = "heap-handlevalue-wrapper" }
```

Comment:
@jdm @sagudev @gterzian Thank you for the review. I’ve addressed all the comments, please take another look. Also, please check https://github.com/servo/mozjs/pull/594, which makes Heap require must_root, and is needed for this PR.